eval/http.go

```diff
@@ -290,7 +292,45 @@ func ApplyResponseContext(ctx context.Context, body hcl.Body, beresp *http.Respo
 		return nil
 	}
 
-	return ApplyResponseHeaderOps(ctx, body, beresp.Header)
+	if err := ApplyResponseHeaderOps(ctx, body, beresp.Header); err != nil {
+		return err
+	}
+
+	content, _, _ := body.PartialContent(config.BackendInlineSchema)
+	if attr, ok := content.Attributes["set_response_status"]; ok {
+		var httpCtx *hcl.EvalContext
+		if c, ok := ctx.Value(ContextType).(*Context); ok {
+			httpCtx = c.eval
+		}
+
+		val, attrDiags := attr.Expr.Value(httpCtx)
+		if seetie.SetSeverityLevel(attrDiags).HasErrors() {
+			return attrDiags
+		}
+
+		status := seetie.ValueToInt(val)
+		if status < 100 || status > 599 {
+			return errors.Configuration.With(
+				fmt.Errorf(
+					"set_response_status sets an invalid HTTP status code: %d; set the status code to 500",
+					status,
+				))
+		}
+
+		if status == 204 {
+			beresp.Request.Context().Value(request.LogEntry).(*logrus.Entry).Warn(
```

Comment:
I am prefer this format, break on method instead of just lines with one `)`
```suggestion
			beresp.Request.Context().
				Value(request.LogEntry).(*logrus.Entry).
				Warn("set_response_status: removing body, if any due to status-code 204")
```